---
float32 pos_x  # x-coordinate of next position 
float32 pos_y  # y-coordinate of next position 

 
 
 